### PR TITLE
feat: added step form

### PR DESCRIPTION
**Pull request by prabhat-wednesday**
### Ticket Link

---

### Related Links

---

### Description
- UI Steps 
---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's
Screenshot 2022-11-17 at 7 30 20 PM

---

### CODE REVIEW

File: app/containers/LoginContainer/index.tsx [hunk 1]
```diff
@@ -35,21 +38,39 @@ const FormInputArray = [
...
+
+  const handleChange = (value: any) => {
+    const step = list.indexOf(Object.keys(value)[0]);
+    console.log(step);
+    setCurrentIndex(step);
+  };
   return (
     <Container>
+      <CustomStep current={currentIndex}>
+        <Step title="Finished" description={description} />
```
Comment (christinittyjohn) [inline app/containers/LoginContainer/index.tsx]: Please use translations.

File: app/containers/LoginContainer/index.tsx [hunk 2]
```diff
@@ -35,21 +38,39 @@ const FormInputArray = [
...
+  const { Step } = Steps;
+
+  const handleChange = (value: any) => {
+    const step = list.indexOf(Object.keys(value)[0]);
+    console.log(step);
+    setCurrentIndex(step);
+  };
   return (
     <Container>
+      <CustomStep current={currentIndex}>
```
Comment (christinittyjohn) [inline app/containers/LoginContainer/index.tsx]: Please create a step form component

File: app/containers/LoginContainer/index.tsx [hunk 3]
```diff
@@ -35,21 +38,39 @@ const FormInputArray = [
...
+        <Step title="Finished" description={description} />
+        <Step title="In Progress" description={description} />
+        <Step title="Waiting" description={description} />
+      </CustomStep>
       <FormContainer>
         <Form
+          layout="vertical"
```
Comment (christinittyjohn) [inline app/containers/LoginContainer/index.tsx]: we need a form for each step

**Review by christinittyjohn — CHANGES_REQUESTED** [review]

File: app/components/DynamicForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  formInput: StepperLoginFormInputInterface;
+  handleFormChange: (value: FormInstance<any>) => void;
+  handleSubmit: (value: any) => void;
+}
+
+const DynamicStepForm = ({ formInput, handleSubmit, handleFormChange }: DynamicStepFormProps) => {
+  const [form] = Form.useForm();
+
+  useEffect(() => {
+    handleFormChange(form);
```
Comment (christinittyjohn) [inline app/components/DynamicForm/index.tsx]: What are we doing here?

Comment (prabhat-wednesday) [reply app/components/DynamicForm/index.tsx]: so we are sending that form instance from child to parent

**Pull request merged**